scipy/special/_agm.pxd

```diff
@@ -0,0 +1,73 @@
+
+import cython
+
+from libc.math cimport log, exp, fabs, sqrt, isnan, isinf
```

Comment:
Oh shoot, missed this earlier--want to use the versions of `isnan` etc. in `_c99compat.h` because of https://github.com/scipy/scipy/pull/6653#discussion_r82275237.